### PR TITLE
Wallet from stdin

### PR DESCRIPTION
**Pull request by roman-khimov**

### CODE REVIEW

File: docs/cli.md [hunk 1]
```diff
@@ -83,7 +83,8 @@ see [compiler documentation](compiler.md).
 deploy` and `contract invokefunction`). Wallet management (creating wallet,
 adding addresses/keys to it) is available there as well as wallet-related
 functions like NEP-17 transfers, NEO votes, multi-signature signing and other
-things.
+things. For all commands requiring read-only wallet (like `dump-keys`) a
```
Comment (AnnaShaleva) [inline docs/cli.md:86]: Don't we want to specify this behaviour inside CLI `-w` flag description also?

File: cli/wallet/wallet.go [hunk 2]
```diff
@@ -332,7 +335,7 @@ func addAccount(ctx *cli.Context) error {
 }
 
 func exportKeys(ctx *cli.Context) error {
-	wall, err := openWallet(ctx.String("wallet"))
+	wall, err := readWallet(ctx.String("wallet"))
```
Comment (AnnaShaleva) [inline cli/wallet/wallet.go:338]: We also have several other usages of `openWallet`, e.g. inside `import multisig`, `import deployed` and etc. Don't we want to replace them also?

Comment (roman-khimov) [reply cli/wallet/wallet.go:338]: No, we don't, they can't be replaced because they change the wallet.

File: docs/cli.md (hunk 1 above)
Comment (roman-khimov) [reply docs/cli.md:86]: Fixed.

**Review by AnnaShaleva — APPROVED** [review]

**Pull request merged**